.expeditor/verify.pipeline.yml

```diff
@@ -1,10 +1,8 @@
 steps:
   - label: ":linux: :ruby:"
     commands:
-      - ruby --version
-      - bundler --version
-      - bundle install
-      - bundle exec rake
+      - chmod +x /workdir/.expeditor/buildkite/verify.sh
+      - /workdir/.expeditor/buildkite/verify.sh
```

Comment:
usually we have this as `.expeditor/buildkite/verify.sh` - not a required change but that's how most projects do it